lib/node_modules/@stdlib/array/complex64/docs/types/index.d.ts

```diff
@@ -761,6 +761,37 @@ declare class Complex64Array implements Complex64ArrayInterface {
 	* // returns true
 	*/
 	some<U = unknown>( predicate: Predicate, thisArg?: ThisParameterType<Predicate<U>> ): boolean;
+
+	/**
+	* Returns subarray elements.
+	*
+	* @param begin - starting index (inclusive)
+	* @param end - ending index (non-inclusive)
+	* @throws indices must be integers
+	* @returns subarray
+	*
+	* @example
+	* var arr = new Complex64Array( 5 );
+	*
+	* arr.set( [ 1.0, -1.0 ], 0 );
+	* arr.set( [ 2.0, -2.0 ], 1 );
+	* arr.set( [ 3.0, -3.0 ], 2 );
+	* arr.set( [ 4.0, -4.0 ], 3 );
+	* arr.set( [ 5.0, -5.0 ], 4 );
+	*
+	* var subarr = arr.subarray();
+	* // returns <Complex64Array>
+	*
+	* var len = subarr.length;
+	* // returns 5
```

Comment:
Same comment as README. I'd extend the example a bit by retrieving the first and last subarray elements here and below.